Update rebuild amount

trofi opened the pull request:
###### Motivation for this change

Improve default help message to have a good executable example.

###### Things done

- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

trofi committed:
maintainers/scripts/rebuild-amount.sh: report parallelism setting

This slightly helps discovering parallelism support.

trofi committed:
maintainers/scripts/rebuild-amount.sh: add basic example

It's not clear if script should be used against HEAD or HEAD^.
Let's have a copyable example.

r-burns commented:
Looks like ofborg is complaining about the tabs in this file. Would you mind either converting to spaces, or adding an editorconfig entry for it with `indent_style = tab`?

trofi committed:
maintainers/scripts/rebuild-amount.sh: convert tabs to four spaces

One notable change is here-document conversion.
Before:

  <TAB>cat <<-FOO
  <TAB>...
  <TAB>FOO

After:

  <sp><sp><sp><sp>cat <<FOO
  <sp><sp><sp><sp>...
  FOO

Note seemingly misaligned 'FOO': '-FOO' understands leading tabs,
but not spaces.

trofi commented:
Sounds good! Added a separate commit to convert tabs to 4-spaces (had to do a minor tweak around here-document to keep it working).

r-burns commented on maintainers/scripts/rebuild-amount.sh:
```diff
@@ -4,123 +4,130 @@ set -e
+            | grep -v '^\(nix-repl.*\)\?$' | tail -n 1 || true)"
 
-echo "Estimating rebuild amount by counting changed Hydra jobs."
+echo "Estimating rebuild amount by counting changed Hydra jobs (parallel=$parallel)."
```
```suggestion
echo "Estimating rebuild amount by counting changed Hydra jobs (parallel=${parallel:-unset})."
```
I think this is a bit more descriptive when parallel is not set in the config, than `(parallel=)`

trofi committed:
Update maintainers/scripts/rebuild-amount.sh

Co-authored-by: Ryan Burns <52847440+r-burns@users.noreply.github.com>

r-burns reviewed (approved):
Looking good, thanks!
